test/NuGet.Clients.Tests/NuGet.CommandLine.Test/RestoreNETCoreTest.cs

```diff
@@ -9275,7 +9275,7 @@ public async Task RestoreNetCore_CPVMProject_DirectDependencyCentralVersionChang
 
                 // Expect exit code 1 on this restore
                 r = Util.RestoreSolution(pathContext, 1);
-                Assert.True(r.AllOutput.Contains("NU1004: The packages lock file is inconsistent with the project dependencies so restore can't be run in locked mode. Disable the RestoreLockedMode MSBuild property or pass an explicit --force-evaluate option to run restore to update the lock file."));
+                Assert.True(r.AllOutput.Contains("NU1004: The package reference x version has changed from [1.0.0, ) to [2.0.0, )."));
```

Comment:
This is a direct reference, correct?

Reply:
That is correct. The test name is `RestoreNetCore_CPVMProject_DirectDependencyCentralVersionChanged_FailsRestore`.